modules/river/hm.nix

```diff
@@ -5,11 +5,17 @@
     config.lib.stylix.mkEnableTarget "River" true;
 
   config = lib.mkIf (config.stylix.enable && config.stylix.targets.river.enable) {
-    wayland.windowManager.river.settings = {
-      border-color-focused = "0x${config.lib.stylix.colors.base0D}";
-      border-color-unfocused = "0x${config.lib.stylix.colors.base03}";
-      border-color-urgent = "0x${config.lib.stylix.colors.base08}";
-      xcursor-theme = config.stylix.cursor.name;
-    };
+    wayland.windowManager.river.settings =
+      let
+        inherit (config.lib.stylix) colors;
+        inherit (config.stylix) cursor;
+      in
+      {
+        border-color-focused = "0x${colors.base0D}";
+        border-color-unfocused = "0x${colors.base03}";
+        border-color-urgent = "0x${colors.base08}";
+        background-color = "0x${colors.base00}";
+        xcursor-theme = "${cursor.name} ${toString cursor.size}";
```

Comment:
Where is this option documented? I can't find it with a search engine.

Just wondering because when using environment variables, `XCURSOR_THEME` and `XCURSOR_SIZE` are separate.

Reply:
Thanks. Looks like this usage is correct.